PolarisO365.ps1

```diff
@@ -820,170 +821,130 @@ function Get-PolarisO365SharePoint() {
-
-    $node_array = @()
+    $o365Sites = $null
+    $o365SharepointDrives = $null
 
     $payload = @{
-        "operationName" = "O365SharepointObjectQuery";
-        "query" = "query O365SharepointObjectQuery(`$after: String, `$orgId: UUID!, `$filter: [Filter!]!, `$first: Int!, `$sortBy: HierarchySortByField, `$sortOrder: HierarchySortOrder) {
-            o365SharepointObjects(after: `$after, fid: `$orgId, filter: `$filter, first: `$first, sortBy: `$sortBy, sortOrder: `$sortOrder) {
-                edges {
-                    node {
-                        objectId
-                        parentId
-                        title
-                        ... on O365Site {
-                            logicalPath {
-                                fid
-                                name
-                            }
-                            effectiveSlaDomain {
-                                id
-                                name
-                                ... on ClusterSlaDomain {
-                                    cluster {
-                                        id
-                                        name
-                                    }
-                                }
-                            }
-                            objectType
-                            slaAssignment
-                        }
-                        ... on O365SharepointDrive {
-                            logicalPath {
-                                fid
-                                name
-                            }
-                            effectiveSlaDomain {
-                                id
-                                name
-                                ... on ClusterSlaDomain {
-                                    cluster {
-                                        id
-                                        name
-                                    }
-                                }
-                            }
-                            objectType
-                            slaAssignment
-                        }
-                    }
-                }
-                pageInfo {
-                    endCursor
-                    hasNextPage
-                    hasPreviousPage
-                }
-            }
-        }";
+        "query"         = "";
         "variables"     = @{
             "after"     = $null;
             "filter"    = @(
                 @{
-                    "field" = "NAME";
+                    "field" = "IS_RELIC";
+                    "texts" = @("false");
+                },
+                @{
+                    "field" = "NAME_OR_EMAIL_ADDRESS";
                     "texts" = @($SearchString);
                 }
-           )
+            );
             "first"     = 100;
-            "orgId"     = $SubscriptionId;
-            "sortBy"    = "NAME";
+            "o365OrgId" = $SubscriptionId;
+            "sortBy"    = "EMAIL_ADDRESS";

```

Comment:
EMAIL_ADDRESS does not apply to SharePoint object type, you should remove it.

Reply:
Removed variables related to sorting